dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -395,11 +395,39 @@ def _check_batch_size(self, classification, input_idx):
             self._pipe._cur_iter_batch_info.check_input(
                 len(classification.data), self._source_context, self._op_name, input_idx)
 
+    def _check_call_arg_meta_data(self, expected_data, actual_data, arg_type, value):
```

Comment:
I wouldn't mind mentioning in the docstring what `arg_type` and `value` should represent.

Reply:
done